validator/valnode/redis/consumer.go

```diff
@@ -139,6 +170,8 @@ var DefaultValidationServerConfig = ValidationServerConfig{
 	ConsumerConfig: pubsub.DefaultConsumerConfig,
 	ModuleRoots:    []string{},
 	StreamTimeout:  10 * time.Minute,
+	Workers:        1,
```

Comment:
Why default to 1 here? Most nitro things use as many cores as is available, so I think defaulting to 0 / runtime.NumCPU() would be a better fit.

Reply:
wanted to make default keep current behavior, but 0 might make sens

Reply:
We're probably the only team using this right now, so I think changing behavior is fine. We can document it in the release notes too.